Makefile

```diff
@@ -122,6 +122,13 @@ catalog: operator-source
 		./scripts/gen_operator_csv.py $$DIR $$OPERATOR_NAME $$OPERATOR_NAMESPACE $$OPERATOR_VERSION $(IMAGE_REGISTRY)/$(IMAGE_REPOSITORY)/$$OPERATOR_NAME:v$$OPERATOR_VERSION $(CHANNEL) || (echo "Failed to generate, cleaning up catalog-manifests/$$OPERATOR_NAME/$$OPERATOR_VERSION" && rm -rf catalog-manifests/$$OPERATOR_NAME/$$OPERATOR_VERSION && exit 3); \
 	done
 
+.PHONY: check-operator-images
+check-operator-images: operator-source
```

Comment:
It would be cool if we could iterate over `$(OPERATORS)` list here, but I think that requires a change to all the operators' Makefiles to change their `env` targets to spit out the image registry URI.

Reply:
It could be passed explicitly from `make env`. Would rather deal with separately if that's something we want because of dependency on the operator changes.

Reply:
It's almost worth it, especially if the image registry or organizational bits are different ... 🤔  hrmm...

Reply:
https://github.com/openshift/dedicated-admin-operator/pull/27

Reply:
I would like to merge this PR before adding in update to use `OPERATOR_IMAGE_URI`.  https://github.com/openshift/osd-operators-registry/issues/8

Reply:
Sounds reasonable.

/lgtm